### PR TITLE
I forgot about the mDNS options!

### PR DESCRIPTION
**Pull request by pkendall64**
When I did the work on the unified targets I completely forgot about the mDNS options value.
This brings it back, but puts it into the wifi code where it really belongs.

### CODE REVIEW

**Review by AlessandroAU — APPROVED** [review]

**Comment by CapnBry** [comment]
TL;DR: PR looks perfect, but I think UID should be removed.

This make sense, but the Configurator currently uses `DMY_BINDING_PHRASE` to fill the associated field. Broadcasting this information never seemed like a good idea to me, but I figured convenience kinda makes it OK.

Now, we don't know for sure what the binding phrase is, so I see you're broadcasting the UID instead. This can't fill the Configurator field although it could be used to flash the device if it were supported both in our build system and the Configurator. It would have to switch between allowing bind phrase _or_ UID bytes and the build system would need to accept UID bytes in preference to a bind phrase. It is starting to get messy compared to the benefit.

If someone is in the Configurator flashing their gear, we should expect that they already have a bind phrase typed in which was saved from their last use. There's no reason to broadcast the bind phrase or the UID bytes to override what should already be filled in.

**Comment by pkendall64** [comment]
Maybe I should just change the json to include the binding phrase as well? And change the mDNS to send that back?

**Review by CapnBry — APPROVED** [review]
Hey the opts are back, baby! I do think it was best to just remove both UID and bind phrase, since the Configurator should be saving that anyway so it doesn't need to be broadcast by every RX the user owns.
